### DISCUSSION TITLE
想问一下git lfs拉取的文件可以在那里直接下载吗

### DISCUSSION THREAD

**Discussion by jackTJT** [Tencent/libpag#2089] [General] [answered]
想问一下git lfs拉取的文件可以在那里直接下载吗，可能会直接下载压缩包来避开网络问题，而直接下载压缩包时，会因为没有git lfs，而导致linux下的bash脚本运行出错

**Comment by kevingpqi123** [comment, marked as answer]
都在源码工程中，你可以在源码页面找到对应的文件直接点击下载，然后去做替换
